service/test/integration2_test.go

```diff
@@ -1708,6 +1747,9 @@ func (c *brokenRPCClient) call(method string, args, reply interface{}) error {
 }
 
 func TestUnknownMethodCall(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
This test simply checks that we don't stall on unknown API calls. It's very strange that it wouldn't work on arm64 given that it's completely architecture independent.

Reply:
I thought it was a test for func call in debug program execute before, so i skiped it. But I still think it should be skiped for arm64-delve check until remote debug released on arm64.

Reply:
There's no difference between local and remote debugging with delve.

Reply:
Okey, i will check later.

Reply:
Okey, i will check later.

Reply:
the skip-code removed.